manifests/1.3.5/opensearch-1.3.5.yml

```diff
@@ -0,0 +1,15 @@
+---
+schema-version: '1.0'
+build:
+  name: OpenSearch
+  version: 1.3.5
+ci:
+  image:
+    name: opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2
```

Comment:
Needs java args

Reply:
Since this was auto-PRed will you add it? ;)

Reply:
Done! Please review! Thanks!

Reply:
I don't see it. Push upstream?

Reply:
Looks like the workflow ran again and removed my commit. https://github.com/opensearch-project/opensearch-build/pull/2340/commits/98f523dd320aefcaa5b2bfc034cfbe68b46d81c9

Reply:
Added now!